src/pages/Top/index.tsx

```diff
@@ -1,26 +1,47 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { useLocation } from 'react-router-dom'
 import { css } from '@emotion/react'
 import { colorToRgb } from '/@/utils/color'
 import { card } from '/@/utils/card'
+import api from '/@/utils/apis'
 
 import PlayerName from './PlayerName'
 import AvatarSelect from './AvatarSelect'
 
 // TODO:クエリがある場合は，招待リンクを踏んだパターンとして表示を変える
 // 招待リンク ?c=xxxxxroomIdxxxxx
 const Top = () => {
-  // TODO: アバター
+  const search = useLocation().search
```

Comment:
嘘かもしれんけど他のとこ分割代入使ってた気がするから使ったほうがいい:kamomushi_fast:
```suggestion
  const { search } = useLocation()
```